Commented the changes

ushukkla opened the pull request:
can we use these changes

ushukla116 committed:
Commented the changes

can we use these changes

nishnik commented:
I don't think so, the current method is simple.
If you believe the method you have commented is better, could you provide arguments. It is a good practice to back your changes with good arguments regarding it.

ushukla116 committed:
Changing the design of the text.

Changed the color of the Pop Up message.

ushukla116 committed:
Designing of popup messages

Added some design on the messages.

ushukla116 committed:
Can we include changes for Youtube kids as well?

Included the URL of Youtube kids if anyone intrested, can also get to it.

OrkoHunter commented:
@Manu-shukla I find it very funny when you write interrogative commit messages. The commit messages are a declarative/imperative summary of the change you are committing. Use the GitHub issues and comments to discuss things and changes before (preferred) or after committing. Also, please read more on "How to write good commit messages?"

ushukla116 committed:
Created Content.js.

Created new file Content.js for script injecting on every site other than YouTube. That means if any user clicks on the Play Next extension when he/she haven't visited the YouTube site. Then he/she may get a message that "Visit YouTube Site to make your own Playlist".

ushukla116 committed:
Delete conten.js
